### PR TITLE
Mobile Games Library

### PR DESCRIPTION
**Pull request by Turqut7**
Project

### CODE REVIEW

File: turqut/Mobile Games Library first-app/src/App.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    scanner.close();
+                    System.exit(0);
+
+                default:
+                    System.out.println("Invalid option. Please try again.");
+            }
+        }
+    }
+
+    private static int getUserOption(Scanner scanner) {
```
Comment (sabuhihuseynov) [inline turqut/Mobile Games Library first-app/src/App.java:96]: do we need this method? If user will insert wrong input we print `Invalid option. Please try again.` in switch case default. I think this method is redundant

File: turqut/Mobile Games Library first-app/src/service/impl/MobileGamesLibraryImpl.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+package service.impl;
+
+import model.Game;
+import model.GameCategory;
+import service.GameLibrary;
+
+public class MobileGamesLibraryImpl implements GameLibrary {
+    private Game[][] games;
```
Comment (sabuhihuseynov) [inline turqut/Mobile Games Library first-app/src/service/impl/MobileGamesLibraryImpl.java:8]: why we need multidimensional array?

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

File: turqut/Mobile Games Library first-app/src/App.java (hunk 1 above)
Comment (Turqut7) [reply turqut/Mobile Games Library first-app/src/App.java:96]: if user enter non-integer input we will meet inputmissmatch exception. And this method handle exception

File: turqut/Mobile Games Library first-app/src/service/impl/MobileGamesLibraryImpl.java (hunk 2 above)
Comment (Turqut7) [reply turqut/Mobile Games Library first-app/src/service/impl/MobileGamesLibraryImpl.java:8]: Using a two-dimensional games array allows each game category to be stored in a separate array. This design makes it easy to manage the games of each category separately. One column for each game category and each column contains games from that category. When we use display all games option, firstofall we show category name with letter case and then games. That's why use two dimensional array.

File: turqut/Mobile Games Library first-app/src/App.java (hunk 1 above)
Comment (sabuhihuseynov) [reply turqut/Mobile Games Library first-app/src/App.java:96]: you can get input as a nextLine then

**Pull request merged**